rover_msgs/FootCmd.lcm

```diff
@@ -0,0 +1,6 @@
+package rover_msgs;
+
+struct FootCmd {
+	double sensor; // [-1.0, 1.0]
+    double claw; // [-1.0, 1.0]
```

Comment:
It's unclear what these mean. Please clarify in the comments. What is the meaning of -1 vs 1? Add units to the comment?

Reply:
implemented